src/apps/tpcc/app/tpcc_setup.h

```diff
@@ -131,7 +131,7 @@ namespace tpcc
       {
         bool is_original = selected_rows.find(i) != selected_rows.end();
         Stock s = generate_stock(i, wh_id, is_original);
-        auto stocks = args.tx.rw(tpcc::TpccTables::stocks);
+        auto stocks = args.tx.rw(TpccTables::stocks);
```

Comment:
Nit: I think all of the changes in this file are unnecessary? Or am I missing something?